reservation/build.gradle.kts

```diff
@@ -0,0 +1,3 @@
+val myMainClass = "it.unibo.lss.fcla.MainClassKt"
```

Comment:
Main class was renamed. Please update with "it.unibo.lss.fcla.reservation.InteractiveReservationMicroserviceStarterClassKt"